lib/Models/Catalog/Esri/esriStyleToTableStyle.ts

```diff
@@ -0,0 +1,169 @@
+  esriStyle?: EsriStyle | null,
+  label?: string | undefined
+) {
+  if (!esriStyle) return {};
+
+  return {
+    // For esriPMS - just use white color
+    // This is so marker icons aren't colored by default
+    color:
+      esriStyle.type === "esriPMS"
+        ? "#FFFFFF"
+        : convertEsriColorToCesiumColor(esriStyle.color)?.toCssColorString(),
+    pointSize: createStratumInstance(TablePointSizeStyleTraits, {}),
+    point: createStratumInstance(PointSymbolTraits, {
+      marker:
+        esriStyle.type === "esriPMS"
+          ? `data:${esriStyle.contentType};base64,${esriStyle.imageData}`
+          : convertEsriMarkerToMaki(esriStyle.style),
+      // symbol.size is used by "esriSMS"
+      // height and width is used by "esriPMS"
+      height:
+        convertEsriPointSizeToPixels(esriStyle.size) ??
+        convertEsriPointSizeToPixels(esriStyle.height),
+      width:
+        convertEsriPointSizeToPixels(esriStyle.size) ??
+        convertEsriPointSizeToPixels(esriStyle.width),
+      rotation: esriStyle.angle,
+      pixelOffset: [esriStyle.xoffset ?? 0, esriStyle.yoffset ?? 0],
+      legendTitle: label || undefined
+    }),
+    outline:
+      esriStyle.outline?.style !== "esriSLSNull"
+        ? createStratumInstance(OutlineSymbolTraits, {
+            color:
+              esriStyle.type === "esriSLS"
+                ? convertEsriColorToCesiumColor(
+                    esriStyle.color
+                  )?.toCssColorString()
+                : convertEsriColorToCesiumColor(
+                    esriStyle.outline?.color
+                  )?.toCssColorString(),
+            // Use width if Line style
+            width:
+              esriStyle.type === "esriSLS"
+                ? convertEsriPointSizeToPixels(esriStyle.width)
+                : convertEsriPointSizeToPixels(esriStyle.outline?.width),
+            legendTitle: label || undefined,
+            dash:
+              esriStyle.type === "esriSLS"
+                ? convertEsriLineStyleToDashArray(esriStyle.style)
+                : convertEsriLineStyleToDashArray(esriStyle.outline?.style)
+          })
+        : undefined
+  };
+}
+
+function convertEsriMarkerToMaki(
+  esri: SupportedEsriSimpleMarkerStyle | string | undefined
+): string {
+  switch (esri) {
+    case "esriSMSCross":
+      return "hospital";
+    case "esriSMSDiamond":
+      return "diamond";
+    case "esriSMSSquare":
+      return "square";
+    case "esriSMSTriangle":
+      return "triangle";
+    case "esriSMSX":
+      return "cross";
+    case "esriSMSCircle":
+    default:
+      return "point";
+  }
+}
+
+// ESRI uses points for styling while cesium uses pixels
+export function convertEsriPointSizeToPixels(pointSize?: number) {
+  if (!isDefined(pointSize)) return undefined;
+  // 1 px = 0.75 point
+  // 1 point = 4/3 point
+  return (pointSize * 4) / 3;
+}
+
+function convertEsriColorToCesiumColor(
+  esriColor?: null | number[] | undefined
```

Comment:
Same comment here about potentially forgetting the parameter without compiler errors.